l10n_de_website_sale_accept_terms/views/views.xml

```diff
@@ -0,0 +1,312 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+
+        <!-- Add needed CSS and JS in template -->
+        <template id="assets_frontend" inherit_id="website.assets_frontend" name="Accept Terms on Checkout">
+          <xpath expr="." position="inside">
+                <link rel='stylesheet' href='/l10n_de_website_sale_accept_terms/static/src/css/style.css'/>
+                <script type="text/javascript" src="/l10n_de_website_sale_accept_terms/static/src/js/script.js"></script>
+          </xpath>
+        </template>
+
+        <!-- Replace the payment-site -->
```

Comment:
Hi @vilder you are right. 

Normaly extending is much better, but we have some restricts by law in germany that nothing must stand between the total amount and the payment button. That's the reason why I replaced it.

I hope this explaination is ok and if you have a better solution for this, you're welcome to tell.